src/plugins/scene3d/Scene3D.cc

```diff
@@ -1638,6 +1664,26 @@ void RenderWindowItem::wheelEvent(QWheelEvent *_e)
       this->dataPtr->mouseEvent, math::Vector2d(scroll, scroll));
 }
 
+////////////////////////////////////////////////
+void RenderWindowItem::keyPressEvent(QKeyEvent *_event)
+{
+  QKeyEvent *keyEvent = static_cast<QKeyEvent*>(_event);
+  if (keyEvent)
+  {
+    this->HandleKeyPress(keyEvent);
+  }
+}
+
+////////////////////////////////////////////////
+void RenderWindowItem::keyReleaseEvent(QKeyEvent *_event)
+{
+  QKeyEvent *keyEvent = static_cast<QKeyEvent*>(_event);
```

Comment:
Curious why this is needed?  Aren't you static casting to/from the same type here?

Reply:
https://github.com/ignitionrobotics/ign-gui/pull/224/commits/54ec72537a6a7fc75620c22f4019efa6eb39dd17